icu4c/source/test/intltest/regextst.cpp

```diff
@@ -5838,11 +5839,11 @@ void RegexTest::TestBug12884() {
     REGEX_ASSERT(status == U_REGEX_TIME_OUT);
 
     // UText, wrapping non-UTF-16 text, also takes a different execution path.
-    const char *text8 = reinterpret_cast<const char*>(u8"¿Qué es Unicode?  Unicode proporciona un número único para cada"
+    StringPiece text8(u8"¿Qué es Unicode?  Unicode proporciona un número único para cada"
```

Comment:
Nit: You could still use `StringPiece text8 = u8"...";` here, right? There's no _technical_ reason to switch to a direct-initialization with parentheses?

Reply:
That's right, but this here is not a unit test of StringPiece, just a test of something where the input wants to include non-ASCII characters. I don't think it matters whether I use assignment or construction.